src/App.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import Calculator from './components/Calculator';
+
+function App() {
+  return (
+    <Calculator />
+  );
+}
+
```

Comment:
- [ ] Kindly rewrite `App.js` to class based component as well 😉 
- [ ] Something looks to be broken on UI side, kindly check it:
<img width="1116" alt="Screenshot 2022-08-10 at 10 59 25" src="https://user-images.githubusercontent.com/77038610/183835818-fed07ef3-a939-41ea-805c-505bb34620b3.png">